caiman/source_extraction/cnmf/utilities.py

```diff
@@ -1154,9 +1154,9 @@ def fast_graph_Laplacian(mmap_file, dims, max_radius=10, kernel='heat',
         else:
             res = dview.map(fast_graph_Laplacian_pixel, pars, chunksize=128)
         indptr = np.cumsum(np.array([0] + [len(r[0]) for r in res]))
-        indeces = [item for sublist in res for item in sublist[0]]
+        indices = [item for sublist in res for item in sublist[0]]
```

Comment:
I feel a little weird that it's correcting code rather than just comments, although this is a change I'd make myself so let's leave it in